stories/ui/Badge.js

```diff
@@ -6,16 +6,34 @@ import Badge from 'ui/components/Badge';
 
 const label = 'Hello Badge';
 
-storiesOf('Badge', module)
-  .add('default (no type)', () => <Badge label={label} />)
-  .add('experimental type', () => <Badge type="experimental" label={label} />)
-  .add('featured type', () => <Badge type="featured" label={label} />)
-  .add('restart-required type', () => (
-    <Badge type="restart-required" label={label} />
-  ))
-  .add('not-compatible type', () => (
-    <Badge type="not-compatible" label={label} />
-  ))
-  .add('requires-payment type', () => (
-    <Badge type="requires-payment" label={label} />
-  ));
+const sections = [
+  {
+    type: 'featured',
+  },
+  {
+    type: 'experimental',
+  },
+  {
+    type: 'restart-required',
+  },
+  {
+    type: 'not-compatible',
+  },
+  {
+    type: 'requires-payment',
+  },
+];
+
+storiesOf('Badge', module).addWithChapters('Badge variations', {
+  useTheme: true,
+  chapters: [
+    {
+      sections: sections.map((section) => {
+        return {
+          subtitle: `type=${section.type}`,
+          sectionFn: () => <Badge {...section} label={label} />,
```

Comment:
It's really important that we get Flow coverage for props because otherwise the stories will drift out of sync quickly. The best idea I have is to add a type hint. Any other ideas? This isn't so bad, though:

```diff
diff --git a/src/ui/components/Badge/index.js b/src/ui/components/Badge/index.js
index f13266b4e..be5608e79 100644
--- a/src/ui/components/Badge/index.js
+++ b/src/ui/components/Badge/index.js
@@ -5,7 +5,7 @@ import Icon from 'ui/components/Icon';
 
 import './styles.scss';
 
-type Props = {|
+export type Props = {|
   label: string,
   type?:
     | 'experimental'
diff --git a/stories/ui/Badge.js b/stories/ui/Badge.js
index 9146ddd40..13f50d727 100644
--- a/stories/ui/Badge.js
+++ b/stories/ui/Badge.js
@@ -3,6 +3,7 @@ import React from 'react';
 import { storiesOf } from '@storybook/react';
 
 import Badge from 'ui/components/Badge';
+import type { Props as BadgeProps } from 'ui/components/Badge';
 
 const label = 'Hello Badge';
 
@@ -29,9 +30,10 @@ storiesOf('Badge', module).addWithChapters('Badge variations', {
   chapters: [
     {
       sections: sections.map((section) => {
+        const props: BadgeProps = { label, ...section };
         return {
           subtitle: `type=${section.type}`,
-          sectionFn: () => <Badge {...section} label={label} />,
+          sectionFn: () => <Badge {...props} />,
         };
       }),
     },

```